app/src/main/AndroidManifest.xml

```diff
@@ -11,9 +11,10 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name="com.heapixLearn.discovery.MainActivity">
+        <activity android:name=".MainActivity">
```

Comment:
@dianaprimshits ui-related stuff shouldn't be in the pr

Reply:
Fixed